pallets/automation-time/src/lib.rs

```diff
@@ -453,28 +453,28 @@ pub mod pallet {
 			weight_left -= update_weight;
 
 			// need to calculate the weight of running just 1 task below.
-			if weight_left < 60_000 {
+			if weight_left < <T as Config>::WeightInfo::run_tasks_many_found(1) {
```

Comment:
not sure this value should actually be:
```suggestion
			if weight_left < (<T as Config>::WeightInfo::run_tasks_many_found(1) + <T as Config>::WeightInfo::read() + <T as Config>::WeightInfo::write()) {
```